diff --git a/Makefile b/Makefile
index 0970126..667ae5e 100644
--- a/Makefile
+++ b/Makefile
@@ -9,12 +9,17 @@ CFLAGS := \
 	-O3 \
 	-Iinclude \
 	-Imaps \
-	-Isounds
+	-Isounds \
+	-I/mingw64/include \
+	-DBUILD_TARGET_WINDOWS \
+	-mwindows
 
 LDFLAGS := \
+	-L/mingw64/lib \
 	-lm \
 	-lSDL2 \
-	-lSDL2_mixer
+	-lSDL2_mixer \
+	-mwindows
 
 SRCS := $(wildcard src/*.c)
 OBJS := $(patsubst src/%,lib/%.o,$(SRCS))
diff --git a/mincbuild.conf b/mincbuild.conf
index dbea5f0..f10cc30 100644
--- a/mincbuild.conf
+++ b/mincbuild.conf
@@ -1,8 +1,8 @@
 # toolchain.
 cc = /usr/bin/gcc
 ld = /usr/bin/gcc
-cflags = -std=c99 -pedantic -O3
-ldflags = NONE
+cflags = -std=c99 -pedantic -O3 -DBUILD_TARGET_WINDOWS -mwindows
+ldflags = -L/mingw64/lib -mwindows
 
 # project.
 src_dir = src
@@ -14,7 +14,7 @@ src_exts = c
 hdr_exts = h
 
 # dependencies.
-incs = maps sounds
+incs = maps sounds /mingw64/include
 libs = m SDL2 SDL2_mixer
 
 # toolchain information.
