diff --git a/Makefile b/Makefile
index cd399bd..21733e7 100644
--- a/Makefile
+++ b/Makefile
@@ -8,12 +8,17 @@ CFLAGS := \
 	-pedantic \
 	-O3 \
 	-Iinclude \
+	-I/mingw64/include \
 	-Imaps \
-	$(shell sdl2-config --cflags)
+	$(shell sdl2-config --cflags) \
+	-DBUILD_TARGET_WINDOWS \
+	-mwindows
 
 LDFLAGS := \
+	-L/mingw64/lib \
 	$(shell sdl2-config --libs) \
-	-lm
+	-lm \
+	-mwindows
 
 SRCS := $(wildcard src/*.c)
 OBJS := $(patsubst src/%,lib/%.o,$(SRCS))
diff --git a/mincbuild.conf b/mincbuild.conf
index 3db996b..a466984 100644
--- a/mincbuild.conf
+++ b/mincbuild.conf
@@ -1,8 +1,8 @@
 # toolchain.
 cc = /usr/bin/gcc
 ld = /usr/bin/gcc
-cflags = -std=c99 -pedantic -O3 $(sdl2-config --cflags)
-ldflags = $(sdl2-config --libs)
+cflags = -std=c99 -pedantic -O3 $(sdl2-config --cflags) -DBUILD_TARGET_WINDOWS -mwindows
+ldflags = -L/mingw64/lib $(sdl2-config --libs) -mwindows
 
 # project.
 src_dir = src
@@ -15,7 +15,7 @@ hdr_exts = h
 
 # dependencies.
 # SDL2 config done via `sdl2-config` in `cflags` and `ldflags`.
-incs = maps
+incs = /mingw64/include maps
 libs = m
 
 # toolchain information.
