diff --git a/Makefile b/Makefile
index ea4b86d..4bc929f 100644
--- a/Makefile
+++ b/Makefile
@@ -9,10 +9,12 @@ CFLAGS := \
 	-O3 \
 	-Iinclude \
 	-Imaps \
-	$(shell sdl2-config --cflags)
+	-I/mingw64/include \
+	$(shell sdl2-config --cflags) \
+	-DBUILD_TARGET_WINDOWS
 
 LDFLAGS := \
-	$(shell sdl2-config --libs) -lm
+	-L/mingw64/lib $(shell sdl2-config --libs) -lm
 
 SRCS := $(wildcard src/*.c)
 OBJS := $(patsubst src/%,lib/%.o,$(SRCS))
diff --git a/mincbuild.conf b/mincbuild.conf
index 3db996b..ad9acdc 100644
--- a/mincbuild.conf
+++ b/mincbuild.conf
@@ -1,8 +1,8 @@
 # toolchain.
 cc = /usr/bin/gcc
 ld = /usr/bin/gcc
-cflags = -std=c99 -pedantic -O3 $(sdl2-config --cflags)
-ldflags = $(sdl2-config --libs)
+cflags = -std=c99 -pedantic -O3 $(sdl2-config --cflags) -DBUILD_TARGET_WINDOWS
+ldflags = -L/mingw64/lib $(sdl2-config --libs)
 
 # project.
 src_dir = src
@@ -15,7 +15,7 @@ hdr_exts = h
 
 # dependencies.
 # SDL2 config done via `sdl2-config` in `cflags` and `ldflags`.
-incs = maps
+incs = maps /mingw64/include
 libs = m
 
 # toolchain information.
