diff --git a/Makefile b/Makefile
index 269e707..12a79c5 100644
--- a/Makefile
+++ b/Makefile
@@ -8,12 +8,17 @@ CFLAGS := \
 	-pedantic \
 	-O3 \
 	-Iinclude \
-	-Imaps
+	-I/mingw64/include \
+	-Imaps \
+	-DBUILD_TARGET_WINDOWS \
+	-mwindows
 
 LDFLAGS := \
+	-L/mingw64/lib \
 	-lm \
 	-lSDL2 \
-	-lSDL2_mixer
+	-lSDL2_mixer \
+	-mwindows
 
 SRCS := $(wildcard src/*.c)
 OBJS := $(patsubst src/%,lib/%.o,$(SRCS))
diff --git a/mincbuild.conf b/mincbuild.conf
index 9408463..cec222e 100644
--- a/mincbuild.conf
+++ b/mincbuild.conf
@@ -1,8 +1,8 @@
 # toolchain.
 cc = /usr/bin/gcc
 ld = /usr/bin/gcc
-cflags = -std=c99 -pedantic -O3
-ldflags = NONE
+cflags = -std=c99 -pedantic -O3 -DBUILD_TARGET_WINDOWS -mwindows
+ldflags = -L/mingw64/lib -mwindows
 
 # project.
 src_dir = src
@@ -14,7 +14,7 @@ src_exts = c
 hdr_exts = h
 
 # dependencies.
-incs = maps
+incs = maps /mingw64/include
 libs = m SDL2 SDL2_mixer
 
 # toolchain information.
@@ -22,6 +22,6 @@ cc_cmd_fmt = %c %f -o %o -c %s %i
 cc_inc_fmt = -I%i
 ld_obj_fmt = %o
 ld_lib_fmt = -l%l
-ld_cmd_fmt = %c -o %b %o %l %f
+ld_cmd_fmt = %c -o %b %o %f %l
 cc_success_rc = 0
 ld_success_rc = 0
