diff --git a/Makefile b/Makefile
index a8895cd..1418ea9 100644
--- a/Makefile
+++ b/Makefile
@@ -8,11 +8,16 @@ CFLAGS := \
 	-pedantic \
 	-O3 \
 	-Iinclude \
-	-Imaps
+	-I/mingw64/include \
+	-Imaps \
+	-DBUILD_TARGET_WINDOWS \
+	-mwindows
 
 LDFLAGS := \
 	-lm \
-	-lSDL2
+	-lSDL2 \
+	-lSDL2_mixer \
+	-mwindows
 
 SRCS := $(wildcard src/*.c)
 OBJS := $(patsubst src/%,lib/%.o,$(SRCS))
diff --git a/mincbuild.conf b/mincbuild.conf
index b012f65..9f58336 100644
--- a/mincbuild.conf
+++ b/mincbuild.conf
@@ -1,8 +1,8 @@
 # toolchain.
 cc = /usr/bin/gcc
 ld = /usr/bin/gcc
-cflags = -std=c99 -pedantic -O3
-ldflags = NONE
+cflags = -std=c99 -pedantic -O3 -DBUILD_TARGET_WINDOWS -mwindows
+ldflags = -L/mingw64/lib -mwindows
 
 # project.
 src_dir = src
@@ -15,13 +15,13 @@ hdr_exts = h
 
 # dependencies.
 incs = maps
-libs = m SDL2
+libs = m SDL2 SDL2_mixer
 
 # toolchain information.
 cc_cmd_fmt = %c %f -o %o -c %s %i
 cc_inc_fmt = -I%i
 ld_obj_fmt = %o
 ld_lib_fmt = -l%l
-ld_cmd_fmt = %c -o %b %o %l %f
+ld_cmd_fmt = %c -o %b %o %f %l
 cc_success_rc = 0
 ld_success_rc = 0
